lttngpy/src/lttngpy/_lttngpy_pybind11.cpp

```diff
@@ -66,6 +67,22 @@ PYBIND11_MODULE(_lttngpy_pybind11, m) {
     py::kw_only(),
     py::arg("session_name"),
     py::arg("url"));
+  m.def(
+    "lttng_create_session_snapshot",
+    &lttng_create_session_snapshot,
+    "Create snapshot session.",
+    py::kw_only(),
+    py::arg("session_name"),
+    py::arg("url"));
+  m.def(
+    "lttng_add_snapshot_output",
+    &lttngpy::lttng_add_snapshot_output,
```

Comment:
I would rename this to `add_snapshot_output`, because the general convention here is that:

1. `lttng_*` functions exposed by `lttngpy` as-is keep the same name with the `lttng_` prefix, e.g., `lttng_destroy_session()`
2. Functions wrapped by a `lttngpy::` C++ function (which is then exposed by `lttngpy`) do _not_ keep the `lttng_` prefix, e.g., `destroy_all_sessions()`

```suggestion
    "add_snapshot_output",
    &lttngpy::add_snapshot_output,
```

Reply:
oh! changed the name of these functions now